src/main/java/com/epam/tc/hw2/trello/utils/UserDataProvider.java

```diff
@@ -0,0 +1,34 @@
+package com.epam.tc.hw2.trello.utils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class UserDataProvider {
+
+    public static final String PROPERTIES_PATH = "src/main/resources/auth.properties";
+    private static final Properties properties = new Properties();
+
+    private UserDataProvider() {
```

Comment:
no need for an empty constructor
pls see the same coment above

Reply:
Done